deployment-aws/terraform/batch.tf

```diff
@@ -153,16 +153,13 @@ data "cloudinit_config" "batch_instance" {
     # install aws-cli v2 and copy the static binary in an easy to find location for bind-mounts into containers
     - curl -s -o /tmp/awscliv2.zip https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip
     - unzip -q /tmp/awscliv2.zip -d /run
-    - /run/aws/install -b /usr/bin
-    # add fetch and run batch helper script
-    - curl -s --output-dir /run -O https://aws-genomics-workflows.s3.amazonaws.com/latest/artifacts/aws-ecs-additions.zip
-    - unzip -q /run/aws-ecs-additions.zip -d /usr/local/bin
-    - chmod +x /usr/local/bin/fetch_and_run.sh
+    - /run/aws/install --install-dir /opt/aws-cli --bin-dir /opt/aws-cli/bin
```

Comment:
just: `/run/aws/install`?

Reply:
if these specific install paths are required by Nextflow or another tool, it'd be great to have a comment for that